src/ventures-for-shared-prosperity/focus-area.njk

```diff
@@ -7,14 +7,114 @@ title: Focus area for ideas42 Venture Studio
 {% block masthead %}
   {{ super() }}
   <div class="hero standard-restricted-width">
-    <div class="g-max-3 g-right">
-      <p class="hero__tag">Where those who wish to focus come</p>
-      <h1 class="hero__title">
-      Focus Area
-      </h1>
+{%include "components/cohort-banner.njk" %}
```

Comment:
Also have an unclosed element somewhere on apply https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-29--ideas42ventures.netlify.app%2Fventures-for-shared-prosperity%2Ffocus-area

Reply:
@tylergaw Same with this one, when I run my local files these don't appear.

Reply:
Fixed!